modules/client-configuration/pages/clients-rh-rhui.adoc

```diff
@@ -0,0 +1,424 @@
+This section contains information about using the {redhat} update infrastructure (RHUI) to register traditional and Salt clients running {rhel} operating systems.
+
+For information about using {redhat} content delivery network (CDN) instead, see xref:client-configuration:clients-rh-cdn.adoc[].
+
+
+[IMPORTANT]
+====
+{rhel} clients are based on {redhat} and are unrelated to {sleses}, RES, {redhat}, or {sles}.
+You are responsible for arranging access to {redhat} base media repositories and {rhela} installation media, as well as connecting {productname} Server to the {redhat} content delivery network.
+You must obtain support from {redhat} for all your {rhela} systems.
+If you do not do this, you might be violating your terms with {redhat}.
+====
+
+
+[IMPORTANT]
+====
+Traditional clients are available on {rhel}{nbsp}6 and 7 only.
+{rhel}{nbsp}8 clients are supported as Salt clients.
+====
+
+
+
+== Import Entitlements and Certificates
+
+{redhat} clients require a {redhat} certificate authority (CA) and entitlement certificate, and an entitlement key.
+
+Entitlement certificates are embedded with expiration dates, which match the length of the support subscription.
+To avoid disruption, you will need to repeat this process at the end of every support subscription period.
+
+{redhat} supply a subscription manager tool to manage subscription assignments.
+It runs locally to track installed products and subscriptions.
+Clients must be registered with the subscription manager to obtain certificates.
+
+{redhat} clients use a URL to replicate repositories.
+The URL will change depending on where the {redhat} client is registered.
+
+{redhat} clients can be registered in three different ways:
+
+* {redhat} content delivery network (CDN) at redhat.com
+* {redhat} Satellite Server
+* {redhat} update infrastructure (RHUI) in the cloud
+
+This guide covers clients registered to {redhat} update infrastructure (RHUI).
+You must have at least one system registered to RHUI, with an authorized subscription for repository content.
+
+For information about using {redhat} content delivery network (CDN) instead, see xref:client-configuration:clients-rh-cdn.adoc[].
+
+
+[IMPORTANT]
+====
+Satellite certificates for client systems require a Satellite server and subscription.
+Clients using Satellite certificates are not supported with {productname} Server.
+====
+
+[WARNING]
+====
+Entitlement certificates are embedded with expiration dates, which match the length of the support subscription.
+To avoid disruption, you will need to repeat this process at the end of every support subscription period.
+====
+
+{redhat} supplies the subscription-manager tool to manage subscription assignments.
+It runs locally on the client system to track installed products and subscriptions.
+Register to redhat.com with subscription-manager, then follow this procedure to obtain certificates.
```

Comment:
This time I'm almost certain this is not true.

Your machine is registered as soon as it is deployed in the cloud, there's no operation needed.

Why the "subscription-manager" utility is available, it seems unused:
```
[root@ip-172-17-17-242 ec2-user]# grep PRETTY /etc/os-release 
PRETTY_NAME="Red Hat Enterprise Linux Server 7.8 (Maipo)"

[root@ip-172-17-17-242 ec2-user]# subscription-manager status
+-------------------------------------------+
   Détails du statut du système
+-------------------------------------------+
Overall Status: Unknown

System Purpose Status: Inconnu
```